Formula/terraform-docs.rb

```diff
@@ -0,0 +1,75 @@
+require "language/go"
+
+class TerraformDocs < Formula
+  desc "Tool to generate documentation from Terraform modules"
+  homepage "https://github.com/segmentio/terraform-docs"
+  url "https://github.com/segmentio/terraform-docs/archive/v0.1.0.tar.gz"
+  sha256 "47e66da75e179e61cde11a785487b8b05970154153c60fc765ef1f93a376abe2"
+  head "https://github.com/segmentio/terraform-docs.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/hashicorp/hcl" do
+    url "https://github.com/hashicorp/hcl.git",
+        :revision => "630949a3c5fa3c613328e1b8256052cbc2327c9b"
+  end
+
+  go_resource "github.com/tj/docopt" do
+    url "https://github.com/tj/docopt.git",
+        :revision => "c8470e45692f168e8b380c5d625327e756d7d0a9"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/segmentio/terraform-docs").install buildpath.children
+    Language::Go.stage_deps resources, buildpath/"src"
+    cd "src/github.com/segmentio/terraform-docs" do
+      system "go", "build", "-o", "terraform-docs"
+
+      bin.install "terraform-docs"
```

Comment:
Please use `system "go", "build", "-o", "#{bin}/terraform-docs"` so you can remove this line.